expression/builtin_time_vec_test.go

```diff
@@ -49,7 +49,9 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 	ast.DateLiteral: {},
 	ast.DateDiff:    {},
 	ast.TimeDiff:    {},
-	ast.DateFormat:  {},
+	ast.DateFormat: {
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETDatetime, types.ETString}},
+	},
```

Comment:
The random string maybe can't get valid formatMask and all result is null

Reply:
Thank's for your advice, I've fixed it.

Reply:
Excuse me, the code works well in my local env, but generate an error in the unit-test, I don't know what it means, so could you please give me some precious help, thank's a lot! 
```
[2019-11-05T06:42:24.145Z] created by google.golang.org/grpc.(*addrConn).connect
[2019-11-05T06:42:24.145Z] 	/home/jenkins/agent/workspace/tidb_ghpr_unit_test/go/pkg/mod/google.golang.org/grpc@v1.23.0/clientconn.go:743 +0xfc
[2019-11-05T06:42:24.145Z] FAIL	github.com/pingcap/tidb/ddl	540.374s
[2019-11-05T06:42:24.145Z] FAIL
script returned exit code 1
```

Reply:
I think there should use `timeFormatGener` instead of `dataStrGener`

Reply:
@710698117 PTAL

Reply:
Ping @710698117 Please address this comment.

Reply:
Thanks for your advice, I've fixed the code, could you please review my code, thank's a lot.